pkg/crd/gen.go

```diff
@@ -201,9 +201,14 @@ func removeDefaultsFromSchemaProps(v *apiextlegacy.JSONSchemaProps) {
 		return
 	}
 
+	if v.Default != nil {
+		fmt.Println("Warning: default unsupported in CRD version v1beta1, v1 required. Removing defaults.")
```

Comment:
this should go to stderr, not stdout:

```suggestion
		fmt.Fprintln(os.Stderr, "Warning: default unsupported in CRD version v1beta1, v1 required. Removing defaults.")
```